spec/src/main/asciidoc/relationship.asciidoc

```diff
@@ -51,7 +51,7 @@ Since this specification depends on CDI and interceptors specifications, fault t
 
 * invocation must be business method invocation as defined in http://docs.jboss.org/cdi/spec/1.2/cdi-spec.html#biz_method[CDI specification^].
 
-* if a method and its containing class don't have any fault tolerance interceptor binding, it won't considered as a fault tolerance operation
+* if a method and its containing class don't have any fault tolerance interceptor binding, it won't considered as a fault tolerance operation.
```

Comment:
it won't **be** considered

Reply:
the word with fault tolerance makes it look like its not a proper name.  if "Fault Tolerance" is meant to be interpreted as a proper name then its fine, but may be we should capitalize it to be a proper name.

Reply:
done.